src/index.html

```diff
@@ -8,9 +8,34 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchBar searchBarBig"
+      data-qa="big">
```

Comment:
check out point 5 from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)